operations/mimir-mixin/alerts/alerts.libsonnet

```
@@ -73,8 +74,8 @@ local utils = import 'mixin-utils/utils.libsonnet';
           },
           annotations: {
             message: |||
-              {{ $labels.job }} {{ $labels.route }} is experiencing {{ printf "%.2f" $value }}s 99th percentile latency.
-            |||,
+              {{ $labels.%s }} {{ $labels.route }} is experiencing {{ printf "%%.2f" $value }}s 99th percentile latency.
+            ||| % $._config.per_job_label,
```

[nit] Could you use the named placeholder for clarity, please?

```suggestion
              {{ $labels.%(per_job_label)s }} {{ $labels.route }} is experiencing {{ printf "%%.2f" $value }}s 99th percentile latency.
            ||| % $._config,
```